frontend/components/Form/_index.html.erb

```diff
@@ -1,4 +1,7 @@
 <form class="Form" action="<%= action %>" accept-charset="UTF-8" method="post">
+  <% if defined?(patch) %>
```

Comment:
`defined?` will be truthy if `patch` is `true` or `false`. To cover for this we could change `if defined?(patch)` to `if defined?(patch) && patch`.